result/egressonlytracking.yaml

```diff
@@ -0,0 +1,290 @@
+components:
+  schemas:
+    EgressOnlyTracking.Metrics.Request:
+      description: >-
+        The container for a egress only tracking metric request.
+      type: object
+      properties:
+        port_names:
+          description: |-
+            Egress only tracking metrics will be retrieved for these port names.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/EgressOnlyTracking/properties/port_name"
+          x-field-uid: 1
+        tagged_metrics:
+          $ref: '#/components/schemas/EgressOnlyTracking.TaggedMetrics.Filter'
+          x-field-uid: 2
+    EgressOnlyTracking.TaggedMetrics.Filter:
+      description: Filter for tagged metrics
+      type: object
+      properties:
+        include:
+          description: Controls inclusion/exclusion of tagged metrics when fetching egress only tracking metrics.
+          type: boolean
+          default: true
+          x-field-uid: 1
+        include_empty_metrics:
+          description: Controls inclusion/exclusion of tagged metrics where each underlying attributes has zero value or absent value.
+          type: boolean
+          default: false
+          x-field-uid: 2
+        metric_names:
+          description: >-
+            The list of metric names that the returned result set will contain.
+            If the list is empty then all metrics will be returned.
+          type: array
+          items:
+            type: string
+            x-enum:
+              frames_tx:
+                x-field-uid: 1
+              frames_rx:
+                x-field-uid: 2
+              bytes_tx:
+                x-field-uid: 3
+              bytes_rx:
+                x-field-uid: 4
+              frames_tx_rate:
+                x-field-uid: 5
+              frames_rx_rate:
+                x-field-uid: 6
+              tx_l1_rate_bps:
+                x-field-uid: 7
+              rx_l1_rate_bps:
+                x-field-uid: 8
+              tx_rate_bytes:
+                x-field-uid: 9
+              rx_rate_bytes:
+                x-field-uid: 10
+              tx_rate_bps:
+                x-field-uid: 11
+              rx_rate_bps:
+                x-field-uid: 12
+              tx_rate_kbps:
+                x-field-uid: 13
+              rx_rate_kbps:
+                x-field-uid: 14
+              tx_rate_mbps:
+                x-field-uid: 15
+              rx_rate_mbps:
+                x-field-uid: 16
+          x-field-uid: 3
+        filters:
+          description: List of filters to selectively fetch tagged metrics with certain tag and corresponding value.
+          type: array
+          items:
+            $ref: '#/components/schemas/EgressOnlyTracking.MetricTag.Filter'
+          x-field-uid: 4
+    EgressOnlyTracking.MetricTag.Filter:
```

Comment:
Why do we have both Metric and Tagged Metric. Isn't this PR for egress only tracking ?

Reply:
Removed tagged metric.